docs/source/markdown/podman-network-create.1.md

```diff
@@ -73,8 +73,9 @@ Set driver specific options.
 
 All drivers accept the `mtu` option. The `mtu` option sets the Maximum Transmission Unit (MTU) and takes an integer value.
 
-Additionally the `bridge` driver supports the following option:
+Additionally the `bridge` driver supports the following options:
 - `vlan`: This option assign VLAN tag and enables vlan\_filtering. Defaults to none.
+- `isolate`: This option isolates the network from communicating outside of itself, dropping all external incoming connections.
```

Comment:
This is not true, it is only blocking traffic between bridge networks with that option enabled. External traffic is allowed.

Also should mention that the default is false.